### ISSUE TITLE
画面出现百叶窗条纹现象

### ISSUE THREAD

**Issue by sxhdroid** [begeekmyfriend/yasea#784]
![image](https://user-images.githubusercontent.com/11678164/95543760-f7ab4400-0a2b-11eb-8212-5e1259fea919.png)
老师！使用master分支，录制和推流都出现这个现象，请问怎么优化？

**Comment by begeekmyfriend** [comment]
这应该是你的设备问题，是手机吗？

**Comment by sxhdroid** [comment]
> 这应该是你的设备问题，是手机吗？你可以切换软编码试试

感谢老师的回复！是RK3399的板子，使用软编码依然有此现象。我单独预览画面和使用MediaMuxer+MediaCodec录制无这个现象。我不清楚要怎么修改了，请老师赐教

**Comment by begeekmyfriend** [comment]
那可能是OpenGL ES驱动的问题，建议用[non-gpuimage](https://github.com/begeekmyfriend/yasea/tree/non-gpuimage)分支试一下，顺便同步一下代码

**Comment by sxhdroid** [comment]
non-gpuimage分支效果要好很多，但也有轻微现象。另外在rk3399的板子上，录制的视频有些无法播放，并不是所有的不能播放，奇怪了。我现在试WSLiveDemo录制无此现象，只是帧率低和无法推流、录制同时进行

**Comment by begeekmyfriend** [comment]
`WSLiveDemo`没用OpenGL ES吧

**Comment by sxhdroid** [comment]
用了OpenGL ES
